ledger/src/shred.rs

```diff
@@ -64,9 +64,7 @@ pub use {
 };
 use {
     self::{shred_code::ShredCode, traits::Shred as _},
-    crate::{
-        blockstore::{self},
-    },
+    crate::blockstore::{self},
```

Comment:
No action required but it is a little odd that `fmt` decided to pick up changes here